src/arch/InputHandler/InputHandler.cpp

```diff
@@ -175,9 +175,8 @@ DriverList InputHandler::m_pDriverList;
 static LocalizedString INPUT_HANDLERS_EMPTY( "Arch", "Input Handlers cannot be empty." );
 void InputHandler::Create( const RString &drivers_, std::vector<InputHandler *> &Add )
 {
-	const RString drivers = drivers_.empty()? RString(DEFAULT_INPUT_DRIVER_LIST):drivers_;
-	std::vector<RString> DriversToTry;
-	split( drivers, ",", DriversToTry, true );
+	const std::vector<RString>& drivers = drivers_.empty() ? GetDefaultInputDriverList() : VectorFromString(drivers_, ',', true);
+	std::vector<RString> DriversToTry(drivers.begin(), drivers.end());
```

Comment:
Do we need to create a copy here actually? We can just use `drivers` directly no?

Reply:
True, this now doesn't make the copy and renames drivers -> driversToTry to meet with the Google style guide.